digital-deck-server/__tests__/playerTests.js

```diff
@@ -29,7 +29,7 @@ beforeAll(() => {
 
```

Comment:
can you add the play card test here as well?

Reply:
Sure. You want me to just grab them from your old PR?

Reply:
yeah, that would be great. You might have to change initial value of playerCard and tableCard though

Reply:
Never mind, I'll add them. I need to add some error checking when handling request. I'll create a PR for that

Reply:
Sounds good